### PR TITLE
Update publish script for Gestalt

### PR DESCRIPTION
**Pull request by pfarejowicz**
Updating the publish script to reflect changes after separation of Gestalt into its own package.

screen shot 2018-03-20 at 5 24 44 pm

### CODE REVIEW

**Comment by christianvuerings** [comment]
Deploy preview for *gestalt* ready!

Built with commit 6ba369e06e4718d93f78246d75faf610ff4b7b0b

https://deploy-preview-94--gestalt.netlify.com

**Comment by pinterest-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Please add a [changelog.md](https://github.com/pinterest/gestalt/blob/master/CHANGELOG.md) entry for your changes.
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,7 +21,6 @@
 
 * Image: Don't show `alt` text when loading the image in FireFox. (#80)(#84)
 * Tabs: Update the background color to be transparent for unselected tabs (#79)
-* Heading / Text / SegmentedControl: Add `title` when `truncate` is set (#82)
```
Comment (pfarejowicz) [inline CHANGELOG.md]: @christianvuerings I think this was in the wrong place, it didn't make it out in my release

Comment (christianvuerings) [reply CHANGELOG.md]: @pfarejowicz Indeed, I'll fix this as part of #93 (new release with only this change)

**Pull request merged**